Modules/posixmodule.c

```diff
@@ -5231,19 +5391,34 @@ os.posix_spawn
     file_actions: object = None
         A sequence of file action tuples.
     /
-
+    *
+    setpgroup: object = None
+        A sequence of flags tuples.
+    resetids: object = None
+        A sequence of flags tuples.
+    setsigmask: object = None
+        A sequence of flags tuples.
+    setsigdef: object = None
+        A sequence of flags tuples.
+    setscheduller: object = None
```

Comment:
_Scheduler_ only has one L.

Reply:
I will implement that in the next commit following Martin design in the issue.